### DISCUSSION TITLE
Link vs anchor tags

### DISCUSSION THREAD

**Discussion by davadams97** [inertiajs/inertia#1721] [Help] [answered]
I am working on an application using Laravel and Inertia.js with Vue.js that requires a user authorization via the 3rd party service's authorization endpoint. 

In my Inertia component, I use the `Link` element that navigates to the Laravel route which redirects to the 3rd party endpoint internally. I was getting CORSs errors until I tried the anchor tag which resolved the issue. 

Can you explain why the `Link` component does not work?

**Comment by RobertBoes** [comment, marked as answer]
See the docs on external redirects: https://inertiajs.com/redirects#external-redirects
The Inertia `<Link>` component makes an XHR request and it expects an Inertia response. So you'd either have to use the external redirect, or if it's just an external URL, use a regular `<a>` tag

**Comment by joydeep-bhowmik** [comment]
use 
```js
return Inertia::location($url);
``` 
as mentioned in the [docs](https://inertiajs.com/redirects)
